lib/microscope/scope.rb

```diff
@@ -26,9 +26,13 @@ def self.inject_scopes(model, fields, _options)
   protected
 
     def blacklisted_fields
-      return [] unless defined? ActiveRecord::AttributeMethods::BLACKLISTED_CLASS_METHODS
```

Comment:
These changes add support for ActiveRecord 6.1 where the constant `BLACKLISTED_CLASS_METHODS` was renamed to `RESTRICTED_CLASS_METHODS`